Tests/Unit/EmogrifierTest.php

```diff
@@ -2067,4 +2067,20 @@ public function emogrifierNotMapsCssToHtmlIfFeatureIsNotEnabled()
             $html
         );
     }
+ 
+    /**
+     * @test
+     */
+    public function emogrifierMapsClassWithLineFeedInAttrib()
+    {
+        $css = '.test {color: red;}';
+        $this->subject->setHtml($this->html5DocumentType . "<html><body><div class='test\n'></div></body></html>");
```

Comment:
Please use double (escaped) quotes for the attribute value so this matches the way HTML attributes are usually written.